src/convertToObject.js

```diff
@@ -7,6 +7,16 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString.split(';').reduce((styles, styleString) => {
+    const updatedStyles = { ...styles };
```

Comment:
[CODE KNOWLEDGE] - Don't use iteration methods to modify an already existing array/object. The `reduce` method should be used without copying the `styles` object in each iteration. Instead, you can directly modify the `styles` accumulator.